docs/src/main/sphinx/connector/hive.rst

```diff
@@ -192,6 +192,10 @@ Hive connector documentation.
     * - ``hive.create-empty-bucket-files``
       - Should empty files be created for buckets that have no data?
       - ``false``
+    * - ``hive.validate-bucketing``
+      - Enables verification that data is in the correct bucket when reading
```

Comment:
Enables validation, not verification (these are [two different things](https://www.geeksforgeeks.org/differences-between-verification-and-validation/)).